web/user/src/pages/v1/purchase/guest/confirmation.vue

```diff
@@ -0,0 +1,523 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia'
+import dayjs from 'dayjs'
+import { useAdressStore } from '~/store/address'
+import { useCheckoutStore } from '~/store/checkout'
+import { useShoppingCartStore } from '~/store/shopping'
+import type { GuestCheckoutAddress, GuestCheckoutRequest } from '~/types/api'
+import { ApiBaseError } from '~/types/exception'
+
+const addressStore = useAdressStore()
+const { address, addressFetchState } = storeToRefs(addressStore)
+
+const shoppingCartStore = useShoppingCartStore()
+const { calcCartResponseItem, availablePaymentSystem } =
+  storeToRefs(shoppingCartStore)
+const {
+  calcCartItemByCoordinatorId,
+  fetchAvailablePaymentOptions,
+  verifyPromotionCode,
+} = shoppingCartStore
+
+const checkoutStore = useCheckoutStore()
+const { guestCheckout } = checkoutStore
+
+const route = useRoute()
+const router = useRouter()
+
+/**
+ * コーディネーターID（クエリパラメータから算出）
+ */
+const coordinatorId = computed<string>(() => {
+  const id = route.query.coordinatorId
+  if (id) {
+    return String(id)
+  } else {
+    return ''
+  }
+})
+
+/**
+ * プロモーションコード（クエリパラメータから算出）
+ */
+const promotionCode = computed<string | undefined>(() => {
+  const code = route.query.promotionCode
+  if (typeof code === 'string') {
+    return code
+  } else {
+    return undefined
+  }
+})
+
+/**
+ * カート番号（クエリパラメータから算出）
+ */
+const cartNumber = computed<number | undefined>(() => {
+  const id = route.query.cartNumber
+  const idNumber = Number(id)
+  if (idNumber === 0) {
+    return undefined
+  }
+  if (isNaN(idNumber)) {
+    return undefined
+  }
+  return idNumber
+})
+
+const guestAddress = addressStore.guestAddress as GuestCheckoutAddress
+const email = addressStore.email as string
```

Comment:
これで参照してしまうとリアクティブ性がなくなってしまうので、`storeToRefs`で参照しないとかと👀

```suggestion
const { guestAddress, email } = storeToRefs(addressStore.guestAddress)
```